core/Cargo.toml

```diff
@@ -34,7 +34,8 @@ std = [
 lock_api = { version = "0.4", features = ["arc_lock"] }
 math = { package = "winter-math", version = "0.10", default-features = false }
 memchr = { version = "2.7", default-features = false }
-miden-crypto = { version = "0.12", default-features = false }
+#miden-crypto = { version = "0.12", default-features = false }
+miden-crypto = { git = "https://github.com/0xPolygonMiden/crypto/", branch ="al-fix-rpo-randcoin", default-features = false }
```

Comment:
I believe this change is already in `miden-crypto` v0.13.

Reply:
Changed